kfdefs/base/jupyterhub/notebook-images/hitchhikers-guide.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: hitchhikers-guide
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url:
+      "https://github.com/operate-first/hitchhikers-guide"
+    opendatahub.io/notebook-image-name:
+      "Hitchhiker's guide"
```

Comment:
Can we change the name to "Hitchhiker's guide to Operate First"? In case we decide to make more guides in the future.

Reply:
Resolved in latest commit.